docs/tempo/website/traceql/_index.md

```diff
@@ -0,0 +1,251 @@
+---
+title: TraceQL
+menuTitle: TraceQL
+description: Learn about TraceQL, Tempo's query language for traces
+weight: 450
+aliases:
+  - /docs/tempo/latest/traceql/
+keywords:
+  - Tempo query language
+  - query language
+  - TraceQL
+---
+
+# TraceQL
+
+Inspired by PromQL and LogQL, TraceQL is a query language designed for selecting traces in Tempo. Currently, TraceQL query can select traces based on the following:
+
+- Span and resource attributes, timing, and duration
+- Basic aggregates: `count()` and `avg()`
+
+For information on where the language is headed, see [future work](architecture).
+The TraceQL language uses similar syntax and semantics as [PromQL](https://grafana.com/blog/2020/02/04/introduction-to-promql-the-prometheus-query-language/) and [LogQL](https://grafana.com/docs/loki/latest/logql/), where possible. 
+
+TraceQL requires Tempo’s Parquet columnar format to be enabled. For information on enabling Parquet, refer to the [Apache Parquet backend](https://grafana.com/docs/tempo/latest/configuration/parquet/) Tempo documentation.
+
+## TraceQL query editor
+
+With Tempo 2.0, you can use the TraceQL query editor in the Tempo data source to build queries and drill-down into result sets. The editor is available in Grafana’s Explore interface. For more information, refer to [TraceQL query editor](query-editor.md).
+
+<p align="center"><img src="assets/query-editor-http-method.png" alt="Query editor showing request for http.method" /></p>
+
+## Construct a TraceQL query
+
+In TraceQL, a query is an expression that is evaluated on one trace at a time. The query is structured as a set of chained expressions (a pipeline). Each expression in the pipeline selects or discards spansets from being included in the results set. For example:
+
+```
+{ span.http.status_code >= 200 && span.http.status_code < 300 } | count() > 2
+```
+
+In this example, the search reduces traces to those spans where: 
+
+* `http.status_code` is in the range of `200` to `299` and 
+* the number of matching spans within a trace is greater than two. 
+
+Queries select sets of spans and filter them through a pipeline of aggregators and conditions. If a spanset is produced after evaluation on a trace, then this spanset (and by extension the trace) is included in the result set of the query.
+
+
+## Selecting spans
+
+TraceQL differentiates between two types of span data: intrinsics, which are fundamental to spans, and attributes, which are customizable key-value pairs. You can use intrinsics and attributes to build filters and select spans.
+
+In TraceQL, curly brackets `{}` always select a set of spans from the current trace. They are commonly paired with a condition to reduce the spans being passed in.
+
+This simple query will be evaluated on every span of every trace, one at a time.
```

Comment:
I think we should remove lines 54 -> 61. It's redundant with information above and below.

Reply:
Will do. Thank you for the suggestion.